### ISSUE TITLE
Data  acquisition problem 

### ISSUE THREAD

**Issue by Kranito** [MadryLab/datamodels-data#3]
Hi
 When I run the command 
aws s3api get-object --bucket madrylab-datamodels  --key fmow/datamodels/train_50pct.pt  --request-payer  requester  train_50pct.pt
it turns out that access denied 
Are there any problems in this query?

**Comment by sung-max** [comment]
Hi!
We only provide the test set datamodels for FMoW [EDITED].
However, if you want to compute them yourself, you can use the raw data provided in the /data subdirectory and use our estimation code: https://github.com/MadryLab/datamodels

**Comment by Kranito** [comment]
Thanks for your reply！😃
The training set data models are exactly what i want，so I use the command mentioned and want to get these data. But failed and the error is “access denied “
Do you mean you only provide the test set datamodels of fmow? Because I downloaded the test set data successfully

**Comment by Kranito** [comment]
> Hi!
> We only provide the train set datamodels for FMoW.
> However, if you want to compute them yourself, you can use the raw data provided in the /data subdirectory and use our estimation code: https://github.com/MadryLab/datamodels

Thanks for your reply！😃
The training set datamodels is exactly what i want，so I use the command mentioned and want to get these data. But failed and the error is “access denied “
Do you mean you only provide the test set datamodels of fmow? Because I downloaded the test set data successfully

**Comment by sung-max** [comment]
Oh oops, sorry for the confusion, yes, we only provide test set datamodels.
There are no train datamodels for FMow uploaded, and that's why you're getting "access denied."

**Comment by Kranito** [comment]
> Oh oops, sorry for the confusion, yes, we only provide test set datamodels.
> There are no train datamodels for FMow uploaded, and that's why you're getting "access denied."

thanks for your help！
